code/index.tsx

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const electron = require('electron');
```

Comment:
Typescript supports require syntax, but recommends using ES6 module syntax.

I believe it would look like: `import * as electron from "electron"`